COPYING.txt

```diff
@@ -1,3 +1,27 @@
+Copyright (c) 2011-2019
```

Comment:
As said in https://github.com/ArduPilot/pymavlink/pull/326#discussion_r294812195:

> There really is no reason to put some names holding copyright here. If copyright names are wanted to be tracked then a CONTRIBUTORS.md file should be created with everyone inside.